windows/installer/build-installer.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+dir='e:/husarnet/tmp/'
+
+version=$(printf %s $(cat ../../core/husarnet_config.h | grep -P '#define HUSARNET_VERSION "\d+.\d+.\d+.\d+"' | cut -d'"' -f2))
```

Comment:
Nitpick, I believe that in `bash-base` there is a line for extracting that info already.